<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
	border-image: url(:/AirTicketData/background1.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;等线&quot;;
background-color: rgb(170, 255, 255);
color: rgb(255, 170, 127);</string>
        </property>
        <property name="text">
         <string>购票中心</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;等线&quot;;
background-color: rgb(170, 255, 255);
color: rgb(255, 170, 127);</string>
        </property>
        <property name="text">
         <string>订单中心</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;等线&quot;;
background-color: rgb(170, 255, 255);
color: rgb(255, 170, 127);</string>
        </property>
        <property name="text">
         <string>用户中心</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="0">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="13" column="0">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>预定</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="3">
         <widget class="QTableWidget" name="looktable">
          <column>
           <property name="text">
            <string>航班号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>票价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>起飞日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>到达日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>起飞时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>飞行距离</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>机型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>起飞城市</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>目的地</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="comboBox_3"/>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="comboBox_4"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>目的地</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>出发城市</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="5">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>月</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QComboBox" name="comboBox_2"/>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>日</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="my_table">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <column>
           <property name="text">
            <string>姓名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>航班号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>始发地</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>目的地</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>起飞日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>起飞时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>机票价格</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>付款状态</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="8" column="1">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>航班号：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="6">
         <widget class="QTableWidget" name="looktable_2">
          <column>
           <property name="text">
            <string>航班号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>票价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>起飞日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>到达日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>起飞时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>飞行距离</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>机型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>起飞城市</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>目的地</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>付款状态</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>订单信息：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>用户的钱：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="M_money">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelchengshi">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>航班起飞城市：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>航班到达城市：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>订单删除</string>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>订单支付</string>
          </property>
         </widget>
        </item>
        <item row="9" column="4">
         <widget class="QPushButton" name="tuikuan_button">
          <property name="text">
           <string>退款</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>航班查询</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelfei">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>航班起飞时间：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">font: 72pt &quot;华文彩云&quot;;</string>
          </property>
          <property name="text">
           <string>欢迎使用航天管理系统</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
